.env.example

```diff
@@ -1,9 +0,0 @@
-DB_HOST=localhost
```

Comment:
We should probably keep the .env.example in the repo in case we need to clone it again or submit the project for review. Can you revert this deletion or maybe we can just make a new one on the next branch?